cfme/tests/cloud/test_providers.py

```diff
@@ -538,6 +539,42 @@ def test_azure_multiple_subscription(appliance, request, soft_assert):
                                                                                      prov_b[0]))
 
 
+@pytest.mark.tier(1)
```

Comment:
should be 3, as 1 is the most important and this case has `caseimportance: low`

Reply:
Case Importance and tier are separate metadata.

Case importance is of low/medium/high/critical, and represents the how frequently the test case should be executed, along with how important it is when it fails.

Tier markers, integers 0, 1, 2, 3, equate to Case Level, which represents the test case layer from unit, function, system, to integration.